src/main/index.js

```diff
@@ -48,6 +50,10 @@ function runApp() {
   app.commandLine.appendSwitch('enable-file-cookies')
   app.commandLine.appendSwitch('ignore-gpu-blacklist')
 
+  // the http cache causes excessive disk usage during video playback
+  // we've got a custom image cache to make up for disabling the http cache
+  app.commandLine.appendSwitch('disable-http-cache')
```

Comment:
Has [`--disk-cache-size=0`](https://www.electronjs.org/docs/latest/api/command-line-switches/#--disk-cache-sizesize) been tried?

Reply:
Yep and it doesn't make a difference to disk usage during video playback. I also tried setting it to 1 which makes it only do 0.1MB/s which is already a lot better than it was, but it doesn't eliminate the problem completely. I suspect that setting it to 0 tells it to just use their default limit.